### DISCUSSION TITLE
reverse proxy synology

### DISCUSSION THREAD

**Discussion by tabs78** [indomus/forum#1528] [GENERALE - Discussioni generiche]
Ciao,
il mio nas synology ha la possibilità di utilizzare un reverse proxy e fornisce anche un ddns.
vorrei usare queste cose su home assistant ma mi sto bloccando.
nel proxy metto come entrata https://miodominio:743 dove 743 è la porta che ho deciso di usare e che ho aperto sul router all'indirizzo del nas. Come destinazione ho messo http://mioip:8123 dove 8123  è la porta di default. 
Qualcosa però è sbagliato perchè non riesco ad accedere da remoto digitando https://miodominio:743

**Comment by snakuzzo** [comment]
Troppi pochi dettagli. "Non riesco ad accedere da remoto" può voler dire tutto o niente.
Quando chiedete aiuto dovete dettagliare di più.
Sei sicuro che puoi avviare il servizio su porte inferiori alla `1024` ?
A livello di router/modem hai configurato bene il portforwarding?
Hai un ip pubblico? Se la risposta è sì, Il tuo FQDN è correttamente associato al tuo indirizzo ip pubblico?
Hai provato a fare una telnet verso il tuo FQDN su quella porta ?
Hai provato a guardare i log di synology?
Hai provato a guardare i log di HA ?
Visto che sei dietro reverse proxy ti sei assicurato di aver valorizzato correttamente la parte dei trusted proxy sotto il blocco http della configurazione di HA ?

Prima di ulteriori richieste di aiuto...verifica tutto quello che ho scritto, altrimenti perdiamo solo tempo.

Grazie.
